android/src/main/java/io/customer/reactnative/sdk/CustomerIOReactNativeInstance.kt

```diff
@@ -21,6 +23,7 @@ object CustomerIOReactNativeInstance {
         environment: Map<String, Any?>,
         configuration: Map<String, Any?>?,
         packageConfig: Map<String, Any?>?,
+        inAppEventListener: InAppEventListener? = null,
```

Comment:
hmm, I thought about this while doing it in flutter but decided to go against it, as the listener shouldn't be part of initialize I think. 

So if we enable in-app, I just trigger the stream of in-app events and if anyone subscribe they can get the events.

Reply:
That's exactly how I did in react too 😅 [We add listener](https://github.com/customerio/customerio-reactnative/pull/89/files#diff-aff95d5bb6f89c287d2d9b5cfe284d37cef0c9508fb8ed82ed9a151a28d8faa1R46-R49) every time if `enableInApp` was true. While the event listener class [skips emitting event to JS](https://github.com/customerio/customerio-reactnative/pull/89/files#diff-d984cf67e776898fba4c755171d320c5a7eb43c30a6608e121201868bc2e8144R40) if no listeners were attached.

Had to pass it as parameter because the listener is React Native module and can only be accessed by classes having `ReactContext`.